package.json

```diff
@@ -51,10 +51,9 @@
     "ajv": "^6.10.2",
     "bunyan": "^1.8.12",
     "coveralls": "^3.0.5",
-    "eslint": "^5.16.0",
-    "eslint-config-wikimedia": "^0.13.1",
-    "eslint-plugin-jsdoc": "^15.8.0",
-    "eslint-plugin-json": "^1.4.0",
+    "eslint": "^6.8.0",
```

Comment:
Unfortunately we can't do that yet. We still need to be able to run this on node6